File: mobile_through_project_final/src/main/java/com/ssafy/cafe/model/dto/LatestOrder.java
package com.ssafy.cafe.model.dto;

import java.util.Date;

import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class LatestOrder {
    String img;
    int orderCnt;
    String userId;
    int orderId;
    String productName;
    Date orderDate;
    char orderCompleted;
    int productPrice;
    String type;
    int totalPrice;
    int productId;
    
    @Builder
    public LatestOrder(String img, int orderCnt, String userId, int orderId, String productName, Date orderDate,
            char orderCompleted, int productPrice, String type, int totalPrice, int productId) {
        super();
        this.img = img;
        this.orderCnt = orderCnt;
        this.userId = userId;
        this.orderId = orderId;
        this.productName = productName;
        this.orderDate = orderDate;
        this.orderCompleted = orderCompleted;
        this.productPrice = productPrice;
        this.type = type;
        this.totalPrice = totalPrice;
        this.productId = productId;
    }
    
}

File: app/src/main/java/com/ssafy/cobaltcoffee/dto/LatestOrder.kt
package com.ssafy.cobaltcoffee.dto

import com.google.gson.annotations.SerializedName
import java.util.*

// o_id 기준으로 분류하고, img는 그 중 하나로 사용하기
data class LatestOrder(
    var img: String,
    var orderCnt: Int,
    var userId: String,
    var orderId: Int,
    var productName: String,
    var orderDate: Date,
    var orderCompleted: Char = 'N',
    var productPrice: Int,
    var type: String,
    var totalPrice: Int = 0,
    var productId: Int = 0
) {
    constructor(): this("", 0, "", 0, "", Date(), ' ', 0, "", 0, 0)
}